### PR TITLE
fixing Trainee Filter

### PR DESCRIPTION
**Pull request by janet-barbie**
# PR Description

This pull request fixes a bug in the filter functionality on the admin dashboard

# Description of tasks that were expected to be completed

Fixing the trainee Filter on the admin dashboard

# How has this been tested?

To test this: Clone the this repostory, cd into it then run the following commands:

npm install : to install dependencies
npm run dev : to start the server
login as an admin
navigate to the trainees page
Filter the trainees using their names
# Screenshots (If appropriate)
Screenshot 2024-09-06 at 14 38 00

# Please check this Checklist before you submit your PR:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] My code generate no warnings
- [x] My test coverage meet the set test coverage threshold
- [x] There are no vulnerabilities
- [x] There are no conflicts with the base branch

### CODE REVIEW

**Review by JacquelineTuyisenge — COMMENTED** [review]
Hello @janet-barbie , PR description is needed!

File: src/components/DataTable.tsx [hunk 1]
```diff
@@ -17,23 +17,28 @@ interface TableData {
   className?: string;
 }
 
-function DataTable({ data, columns, title, loading }: TableData) {
-  // const sortedData = React.useMemo(() => [...data], []);
```
Comment (Bananayosostene) [inline src/components/DataTable.tsx:21]: I thing you have unnecessary changes, you should removed them wherever they are

**Review by shema-surge — APPROVED** [review]

**Review by shebz2023 — APPROVED** [review]

**Review by andremugabo — APPROVED** [review]
LGTM

**Review by Bananayosostene — APPROVED** [review]

**Review by JacquelineTuyisenge — APPROVED** [review]
